sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -186,50 +186,546 @@ async def get_next(next_link=None):
 
         return AsyncItemPaged(get_next, extract_data)
 
+    @distributed_trace_async
+    async def get_repository_properties(self, repository: str, **kwargs: Any) -> RepositoryProperties:
+        """Get the properties of a repository
+
+        :param str repository: Name of the repository
+        :returns: :class:`~azure.containerregistry.RepositoryProperties`
+        :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+        """
+        return RepositoryProperties._from_generated(  # pylint: disable=protected-access
+            await self._client.container_registry.get_properties(repository, **kwargs)
+        )
+
     @distributed_trace
-    def get_repository(self, repository_name: str, **kwargs: Any) -> ContainerRepository:
-        """Get a repository client
+    def list_manifests(self, repository: str, **kwargs: Any) -> AsyncItemPaged[ArtifactManifestProperties]:
```

Comment:
Since this returns manifest properties and not the manifest itself, would it be ok to name it as such?

Reply:
Yes I can change this, I'm doing it in my main PR [here](https://github.com/Azure/azure-sdk-for-python/pull/18667)